README.md

```diff
@@ -406,12 +407,12 @@ It'll read and sort them by timestamp before generating reports.
 
 ```console
 $ vegeta report -inputs="10.0.1.1.bin,10.0.2.1.bin,10.0.3.1.bin"
-Requests      [total, rate]         3600000, 60000.00
-Latencies     [mean, 95, 99, max]   223.340085ms, 326.913687ms, 416.537743ms, 7.788103259s
-Bytes In      [total, mean]         3714690, 3095.57
-Bytes Out     [total, mean]         0, 0.00
-Success       [ratio]               100.0%
-Status Codes  [code:count]          200:3600000
+Requests      [total, rate]             3600000, 60000.00
+Latencies     [mean, 90, 95, 99, max]   223.340085ms, 318.808559ms, 326.913687ms, 416.537743ms, 7.788103259s
```

Comment:
Do you feel the report is still readable with this addition?

Reply:
Yes, IMHO I think it's still readable (except have to check on Windows, where terminal window width isn't much resizable). Without the addition, the line spans more than the conventional 80 characters line width anyhow.

What could help with or without this addition is to reduce the significant digits after the decimal in the latencies (as well as other fields). Do we really need that many digits? 2 to 3 after decimal should suffice, and that will significantly reduce line width.